publisher/image/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.1
 
+RUN apk add --update-cache curl && rm -rf /var/cache/apk/*
```

Comment:
Yes I can do that but then you need to run exactly that command to get the profiler information (the container is not exposing the port where the profiler is listening).